src/test/java/org/openrewrite/java/testing/junit5/JUnit5BestPracticesTest.java

```diff
@@ -194,27 +192,28 @@ void somethingElse() {
     void changeThrowingRunnableToExecutable() {
         //language=java
         rewriteRun(
-          spec -> spec.typeValidationOptions(TypeValidation.none()),
```

Comment:
This is what prompted the change; the rest then came along after seeing failure with 5.9.2.

Reply:
The `5\\.9` was just a test from my side trying to figure this out.